common/src/main/java/earth/terrarium/adastra/common/blocks/LaunchPadBlock.java

```diff
@@ -139,7 +139,7 @@ private void destroy(Level level, BlockPos pos, BlockState state) {
         }
     }
 
-    private BlockPos getController(BlockState state, BlockPos pos) {
```

Comment:
I've made this method public, since I would have copied the logic anyway and it seems to me like it does not have to be private